dev/docker/ranger/Dockerfile

```diff
@@ -31,26 +32,27 @@ RUN wget https://downloads.apache.org/ranger/${RANGER_VERSION}/apache-ranger-${R
     tar zxvf apache-ranger-${RANGER_VERSION}.tar.gz && \
     ln -s apache-ranger-${RANGER_VERSION} apache-ranger && \
     cd apache-ranger && \
-    mvn -pl ${RANGER_PLUGINS},jisql,agents-audit,agents-common,agents-cred,agents-installer,credentialbuilder,embeddedwebserver,security-admin,ranger-util,ranger-plugin-classloader,ranger-tools,distro -am -DskipTests=true compile package
+    mvn -pl security-admin -P ranger-jdk11 -am -DskipTests=true compile package
```

Comment:
If we directly download ranger-admin.tar.gz, Why do we need to compile it from the source code?

Reply:
You're right, I removed this compile-ranger stage.